### PR TITLE
Create GitHub Actins workflow that creates release

### LINKED ISSUE #65: Incorporate GitHub Actions to create new releases
I think the best way to incorporate GitHub Actions is by having a manual workflow.
The manual workflow would have a few questions to fill out the release information.
Have it look something like this:
Screenshot from 2023-02-14 16-59-16

The workflow would build all kernels and buildinits, then upload them to the release.

### PR DESCRIPTION
**Pull request by rluzuriaga**
This workflow runs all the builds (jobs) concurrently on separate Ubuntu machines. Once the jobs all complete successfully, then the `release` job runs to create the release with all the files.

Resolves #65

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+          cd dist
+          sha256sum -c ./*.sha256
+          if [[ $? -ne 0 ]]; then exit 1; fi
+
+      - name: Save distribution files
+        uses: actions/upload-artifact@v3
+        with:
+          name: distribution-files
+          path: dist
+          retention-days: 1
```
Comment (Sebastian-Roth) [inline .github/workflows/create_release.yml:61]: Do I get this right, this is only the temporary "store" for the files to be distributed/uploaded to the official github release? If so, one day is enough for sure.

**Review by Sebastian-Roth — APPROVED** [review]
Though I have never used GH workflows before this looks pretty straight forward. Close to what I am used to from the other build platform. I have looked through this thoroughly and I think it's good to be merged.

**Comment by Sebastian-Roth** [comment]
@rluzuriaga Thanks! You should be able to merge this now. Please give it a go and let me know if you don't have the rights to merge.

File: .github/workflows/create_release.yml (hunk 1 above)
Comment (rluzuriaga) [reply .github/workflows/create_release.yml:61]: @Sebastian-Roth You are correct. Those "artifacts" are uploaded and can be seen in the `Actions` tab once the workflow finishes.
The `retention-days` can be from 1-90 days. This does not affect the files in the release.

**Comment by rluzuriaga** [comment]
> You should be able to merge this now. Please give it a go and let me know if you don't have the rights to merge.

@Sebastian-Roth I don't have the rights to merge.

**Pull request merged**